### DISCUSSION TITLE
Trying to use multiple sources to load a model

### DISCUSSION THREAD

**Discussion by jcuell2** [vladmandic/sdnext#582] [General]
Where would I need to go to so that I can have multiple sources to obtain models from? I have models on my C drive on my laptop and the rest of my models on my USB due to space issues (laptop only has 500gb), and i know Auto allowed me to use "--ckpt-dir=X:\Example". I tried going to System Paths-> Path for directory with Stable diffusion checkpoints -> inserting my main model there being, C:\User\....\models --ckpt-dir=X:\Example, but all that did was crash. I don't really know what to modify to get the rest of my models to load in and would appreciate any help.

**Comment by tkiikt** [comment]
@jcuell2 see this topic: #531 and Currently, recommend that all model files be placed on a USB drive.

**Reply by vladmandic** [reply]
i wanted to add native support for multiple models folders, but too much of legacy code relies on assumption its a single string value.
what you can do is create symlinks (works on both windows and linux) inside main models folder that point to other folders. loader will scan main folder and all subfolders automatically.
